src/CreateUser.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import axios from "axios";
+
+class CreateUser extends React.Component {
+  
+  state = {
+    userName : '',
+    cpf: '',
+    birthDate: '',
+    registrationNumber: '',
+  }
+
+  handleChangeUserName = (event) => {
+    // console.log('event',event.target.value);
+    this.setState({userName: event.target.value});
+  }
+
+  handleChangeCpf = (event) => {
+    // console.log('event',event.target.value);
+    this.setState({cpf: event.target.value});
+  }
+
+  handleChangeBirthDate = (event) => {
+    // console.log('event',event.target.value);
+    this.setState({birthDate: event.target.value});
+  }
+  
+  handleChangeRegistrationNumber = (event) => {
+    // console.log('event',event.target.value);
+    this.setState({registrationNumber: event.target.value});
+  }
+
+ 
+  handleSubmit =  (event) => {
+    event.preventDefault();
+    fetch("https://dog.ceo/api/breeds/image/random");
+    // console.log("this.state", this.state);
+  }
+
+  handleOnClickButton = (event) => {
+    event.preventDefault();
+    const {userName, cpf, birthDate, registrationNumber} = this.state;
+    axios.post("",{userName, cpf, birthDate, registrationNumber});
```

Comment:
to do a HTTP POST with `fetch`, just add the `method` param on options, something like this:
```tsx
const options = {
  method: 'POST'
};
const response = await fetch('https://mysite.com/api/v1/route', options);
```

Reply:
you've already did it before:
https://github.com/IsaComegna/flying-club-front/blob/d6eea30b483ce4238fb39efa1ff301bfdf6c0ef3/src/FlightSubmit.js#L68-L75